File: Assets/Demo/Follow.js
#pragma strict

var target : GameObject;

var xmod : int;
var ymod : int;
var zmod : int;

function Start () {

}

function Update () {
	transform.position.x = target.transform.position.x + xmod;
	transform.position.y = target.transform.position.y + ymod;
	transform.position.z = target.transform.position.z + zmod;
}

File: CasterPollux/Assets/Scripts/follow.cs
using UnityEngine;
using System.Collections;

public class follow : MonoBehaviour {
	
	public bool isfollowing = false;
	
	public int modx;
	public int mody;
	public int modz;
	public GameObject followed;
	
	// Use this for initialization
	void Start () {
		
	}
	
	// Update is called once per frame
	void Update () {
		transform.position = new Vector3(
			followed.transform.position.x + modx, 
			followed.transform.position.y + mody, 
			followed.transform.position.z + modz
		);
	}
}
